hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/terasort/TeraGen.java

```diff
@@ -0,0 +1,319 @@
+/**
```

Comment:
According to the other classes, here we only need to keep the Apache license, right?

Reply:
Both licenses added, one for dell, another for apache.